### DISCUSSION TITLE
Search function in submodules

### DISCUSSION THREAD

**Discussion by anoikob** [omnetpp/omnetpp#1221] [Ideas]
Hey there, would it be possible to implement a search function in the ned editor when it comes to submodules? Using the inet framework as an example: 
![image](https://github.com/user-attachments/assets/78c66969-f82e-47e6-88d8-ca3d2957bad7)

You can see that there are tons of submodules, but if I want to find something specific I have to look through all of them until i find it (or not). It would be a good idea to add something like a search function or a filter that would help us look through them more efficiently.

**Comment by torokati44** [comment]
The little button with the blue and yellow arrows on the top right corner of your screenshot is a substring filtering function! :)

**Reply by anoikob** [reply]
Wow thank you, i did press it, but since the text cursor started blinking next to the folder, I assumed it was a folder rename button. Thanks a lot!

**Reply by torokati44** [reply]
That's fair, the UX is not the greatest here ... there isn't even a tooltip about what that button does... :/

**Reply by torokati44** [reply]
Since GitHub's auto-cross-referencing doesn't seem to work across issues and discussions - I have posted this as a follow-up: https://github.com/omnetpp/omnetpp/issues/1222
